moveit_ros/moveit_servo/src/servo_calcs.cpp

```diff
@@ -618,9 +632,20 @@ bool ServoCalcs::internalServoUpdate(Eigen::ArrayXd& delta_theta,
                         servo_params_.override_velocity_scaling_factor);
 
   // Enforce SRDF position limits, might halt if needed, set prev_vel to 0
-  const auto joints_to_halt = enforcePositionLimits(next_joint_state_);
+  const auto joints_to_halt =
+      enforcePositionLimits(next_joint_state_, servo_params_.joint_limit_margin, joint_model_group_);
+
   if (!joints_to_halt.empty())
   {
+    std::ostringstream joint_names;
+    std::transform(joints_to_halt.cbegin(), std::prev(joints_to_halt.cend()),
+                   std::ostream_iterator<std::string>(joint_names, " "),
+                   [](const auto& joint) { return " '" + joint->getName() + "' "; });
+
+    joint_names << joints_to_halt.back()->getName();
+    RCLCPP_WARN_STREAM_THROTTLE(LOGGER, *node_->get_clock(), ROS_LOG_THROTTLE_PERIOD,
+                                joint_names.str() << " close to a position limit. Halting.");
```

Comment:
Shouldnt this also start with the string "Joints" before showing joint names?

Also this will put 2 spaces after the last joint since the std::ostream stuff will already end with a space

Reply:
Ah you are right.
Fixed it 
Should print this now
```
Joints 'joint1' 'joint2' 'joint3' close to a position limit. Halting.
```